packages/terra-tabs/src/StructuralTabs.module.scss

```diff
@@ -142,18 +142,18 @@
       }
 
       &.is-disabled {
-        background-color: var(--terra-tabs-structural-disabled-hover-background-color, #fff);
+        background-color: var(--terra-tabs-structural-disabled-background-color, #fff);
         background-size: 0 0;
-        color: var(--terra-tabs-structural-disabled-hover-color, rgba(0, 0, 0, 0.25));
+        color: var(--terra-tabs-structural-disabled-color, rgba(28, 31, 33, 0.2));
         cursor: default;
-        opacity: var(--terra-tabs-structural-disabled-hover-opacity, 1);
+        opacity: var(--terra-tabs-structural-disabled-opacity, 1);
```

Comment:
here as well fusion theme and lowlight themes also needs to be updated with these variable names